.github/start-wasm-example/tests/wasm_example.spec.ts

```diff
@@ -0,0 +1,48 @@
+import { test, expect, Page } from '@playwright/test';
+
+test.beforeEach(async ({ page }) => {
+  await page.goto('http://localhost:8000/');
+});
+
+test.describe('WASM example', () => {
+  test('Wait for success', async ({ page }, test_info) => {
+    let start = new Date().getTime();
+
+    let found = false;
+    while (new Date().getTime() - start < 600000) {
```

Comment:
```suggestion
    while (new Date().getTime() - start < 600_000) {
```

Reply:
Also, we don't have any hard time based deadlines on running time of examples - this way of testing makes it reliant on machine performance. Granted, the margin for safety is almost 2 orders of magnitude, so I suppose it's fine.

Reply:
reduced to 5 minutes, and the longest is at 2 minutes for now